tests/unit/graph/test_selector.py

```diff
@@ -82,7 +68,7 @@ def graph():
 
 
 @pytest.fixture
-def manifest(graph):
+def mock_manifest(graph):
     return _get_manifest(graph)
```

Comment:
Should we be more specific here that we are only mocking the graph functionality part?

Reply:
Yea I can change the name to something like `manifest_with_mocked_graph`.